sig-cloud-provider/CHARTER.md

```diff
@@ -1,100 +1,68 @@
-* The collection of user experience reports from Kubernetes operators running on provider subprojects; and the delivery of roadmap information to SIG PM
+SIG Cloud Provider’s mission is to simplify, develop, and maintain cloud provider integrations as extensions, or add-ons, to Kubernetes clusters.
 
-## Organizational Management
+### In scope
 
-* Six months after this charter is first ratified, it MUST be reviewed and re-approved by the SIG in order to evaluate the assumptions made in its initial drafting
-* SIG meets bi-weekly on zoom with agenda in meeting notes.
-    * SHOULD be facilitated by chairs unless delegated to specific Members
-* The SIG MUST make a best effort to provide leadership opportunities to individuals who represent different races, national origins, ethnicities, genders, abilities, sexual preferences, ages, backgrounds, levels of educational achievement, and socioeconomic statuses
+#### Areas of Focus
 
-## Subproject Creation
+- Extension points between Kubernetes and any cloud provider
+- APIs/interfaces for efficiently provisioning/de-provisioning cloud resources (nodes, routes, load balancers, etc)
+- Configuration of cluster components to enable cloud provider integrations
+- Testing and testing frameworks to ensure vendor neutrality across all cloud providers
 
-Each Kubernetes provider will (eventually) be a subproject under SIG Cloud Provider. To add new sub projects (providers), SIG Cloud Provider will maintain an open list of requirements that must be satisfied.
-The current requirements can be seen [here](https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0002-cloud-controller-manager.md#repository-requirements). Each provider subproject is entitled to create 1..N repositories related to cluster turn up or operation on their platform, subject to technical standards set by SIG Cloud Provider.
-Creation of a repository SHOULD follow the KEP process to preserve the motivation for the repository and any additional instructions for how other SIGs (e.g SIG Documentation and SIG Release) should interact with the repository
+#### Code, Binaries and Services
 
-Subprojects that fall under SIG Cloud Provider may also be features in Kubernetes that is requested or needed by all, or at least a large majority of providers. The creation process for these subprojects will follow the usual KEP process.
+- the [common interfaces](https://github.com/kubernetes/cloud-provider/blob/master/cloud.go) consumed by all cloud providers
+- the [cloud-controller-manager](https://github.com/kubernetes/kubernetes/tree/master/cmd/cloud-controller-manager), which acts as the “out-of-tree” cloud provider component for clusters.
+- core controllers (started by the cloud-controller-manager) that interact with cloud provider resources
+- all [cloud provider repositories](https://github.com/kubernetes?utf8=%E2%9C%93&q=cloud-provider-&type=&language=) under the Kubernetes organization
```

Comment:
What aspects of a specific cloud provider's repository does this SIG claim ownership over?

Reply:
This SIG has ownership of the kubernetes/cloud-provider-* repos. 
The intent is to coordinate and provide guidance for changes which effect multiple cloud-providers, for the best interest of the Kubernetes projects.
As an example how/when/where should IP addresses be assigned from the cloud provider to a node.

Reply:
Agreed with Walter. I think as a default the SIG should own the entire repo (delegating responsibility to subprojects) but I don't think this is a hard requirement, it may make sense for other SIGs to own subdirectories of the provider repos where it makes sense.